lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -555,13 +555,19 @@ void TypeChecker::performTypoCorrection(DeclContext *DC, DeclRefKind refKind,
 }
 
 static InFlightDiagnostic
-diagnoseTypoCorrection(TypeChecker &tc, DeclNameLoc loc, ValueDecl *decl) {
+diagnoseTypoCorrection(TypeChecker &tc, DeclNameLoc loc, ValueDecl *decl,
+                       FunctionRefKind refKind) {
+  
+  auto name = refKind == FunctionRefKind::Compound ? decl->getFullName()
+                                                   : decl->getName();
```

Comment:
Here you must add the quotes back or it'll fall any existing typo correction tests that rely on it.